support androidx

Since last topic was closed Ill open a new one - 

Unfortunately issue is not fixed. My CI still fails because of sms_autofill.
Did you try increasing project compileSdkVersion to 28?


```
* What went wrong:
Execution failed for task ':sms_autofill:verifyReleaseResources'.
> java.util.concurrent.ExecutionException: com.android.builder.internal.aapt.v2.Aapt2Exception: Android resource linking failed
  Output:  /Users/vagrant/git/build/sms_autofill/intermediates/res/merged/release/values/values.xml:197: error: resource android:attr/fontVariationSettings not found.
  /Users/vagrant/git/build/sms_autofill/intermediates/res/merged/release/values/values.xml:198: error: resource android:attr/ttcIndex not found.
  error: failed linking references.
  Command: /Users/vagrant/.gradle/caches/transforms-1/files-1.1/aapt2-3.2.1-4818971-osx.jar/e3f1f324de2b8ea5e50a4a207f40f4e0/aapt2-3.2.1-4818971-osx/aapt2 link -I\
          /Users/vagrant/Library/Developer/Xamarin/android-sdk-macosx/platforms/android-27/android.jar\
          --manifest\
          /Users/vagrant/git/build/sms_autofill/intermediates/aapt_friendly_merged_manifests/release/processReleaseManifest/aapt/AndroidManifest.xml\
          -o\
          /var/folders/6q/wgy6jtp12w5gzgm9lzcglpqw0000gn/T/aapt-3803257974810333648-out\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-xhdpi-v4_common_google_signin_btn_text_dark_normal_background.9.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/layout_notification_action_tombstone.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-ur_values-ur.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable_notification_icon_background.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-pa_values-pa.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-pt-rBR_values-pt-rBR.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-as_values-as.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-xxhdpi-v4_googleg_disabled_color_18.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-fi_values-fi.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable_common_google_signin_btn_icon_dark.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-uk_values-uk.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/layout_notification_template_big_media_narrow_custom.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/layout_notification_template_part_time.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-ka_values-ka.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-iw_values-iw.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-mdpi-v4_notify_panel_notification_icon_bg.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-bs_values-bs.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-xhdpi-v4_googleg_standard_color_18.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-mk_values-mk.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable_notification_bg_low.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-in_values-in.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-zu_values-zu.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-mr_values-mr.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-xhdpi-v4_notification_bg_low_normal.9.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-hr_values-hr.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-en-rAU_values-en-rAU.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-sk_values-sk.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable_common_google_signin_btn_text_light.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/layout_notification_action.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-v21_notification_action_background.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-th_values-th.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-sr_values-sr.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-km_values-km.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-af_values-af.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-pt_values-pt.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-hdpi-v4_googleg_standard_color_18.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-es-rUS_values-es-rUS.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-fr-rCA_values-fr-rCA.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-gl_values-gl.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-watch-v20_values-watch-v20.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-watch-v20_common_google_signin_btn_text_disabled.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/layout-v21_notification_action_tombstone.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-gu_values-gu.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-kk_values-kk.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-mdpi-v4_common_google_signin_btn_icon_dark_normal_background.9.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/layout_notification_template_media_custom.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-zh-rHK_values-zh-rHK.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-xhdpi-v4_googleg_disabled_color_18.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-te_values-te.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/layout_notification_template_big_media_narrow.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/layout_notification_template_media.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-ml_values-ml.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-ky_values-ky.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-ar_values-ar.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-xxhdpi-v4_common_google_signin_btn_icon_light_normal_background.9.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-xhdpi-v4_notify_panel_notification_icon_bg.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable_common_google_signin_btn_icon_disabled.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/layout_notification_template_lines_media.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-en-rIN_values-en-rIN.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-da_values-da.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable_common_google_signin_btn_icon_dark_focused.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-hdpi-v4_common_google_signin_btn_text_light_normal_background.9.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/color_common_google_signin_btn_text_dark.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/layout_notification_media_cancel_action.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-am_values-am.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-ms_values-ms.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-zh-rTW_values-zh-rTW.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-xxhdpi-v4_googleg_standard_color_18.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-en-rGB_values-en-rGB.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/layout_notification_media_action.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-hy_values-hy.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-zh-rCN_values-zh-rCN.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-nl_values-nl.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/layout_notification_template_part_chronometer.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-ro_values-ro.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/layout-v21_notification_template_icon_group.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-my_values-my.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-lv_values-lv.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-pl_values-pl.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-xxhdpi-v4_common_google_signin_btn_icon_dark_normal_background.9.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-lo_values-lo.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/layout_notification_template_big_media.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-hdpi-v4_common_google_signin_btn_text_dark_normal_background.9.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values_values.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-bg_values-bg.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-watch-v20_common_google_signin_btn_text_light_normal.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable_common_google_signin_btn_icon_light_normal.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-xxhdpi-v4_common_google_signin_btn_text_light_normal_background.9.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-sl_values-sl.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-v21_values-v21.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-en-rCA_values-en-rCA.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-hu_values-hu.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-hdpi-v4_googleg_disabled_color_18.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-hdpi-v4_common_google_signin_btn_icon_light_normal_background.9.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-mdpi-v4_googleg_standard_color_18.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable_common_google_signin_btn_text_light_focused.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-vi_values-vi.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-en-rXC_values-en-rXC.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-ru_values-ru.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/layout-v21_notification_template_custom_big.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-hdpi-v4_notification_bg_normal.9.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable_common_google_signin_btn_text_disabled.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-xhdpi-v4_common_full_open_on_phone.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-pt-rPT_values-pt-rPT.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-fr_values-fr.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-v24_values-v24.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-mdpi-v4_common_google_signin_btn_icon_light_normal_background.9.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-et_values-et.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-bn_values-bn.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-ca_values-ca.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-xhdpi-v4_notification_bg_low_pressed.9.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-is_values-is.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-hdpi-v4_notification_bg_normal_pressed.9.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-sv_values-sv.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-tl_values-tl.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable_common_google_signin_btn_icon_dark_normal.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-mdpi-v4_common_google_signin_btn_text_light_normal_background.9.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/layout_notification_template_icon_group.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-fa_values-fa.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-uz_values-uz.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/color_common_google_signin_btn_text_light.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-watch-v20_common_google_signin_btn_text_dark_normal.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable_common_google_signin_btn_text_dark_normal.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-ko_values-ko.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-hdpi-v4_notify_panel_notification_icon_bg.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-hdpi-v4_common_full_open_on_phone.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-hi_values-hi.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-hdpi-v4_common_google_signin_btn_icon_dark_normal_background.9.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-mdpi-v4_notification_bg_normal_pressed.9.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-b+sr+Latn_values-b+sr+Latn.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/color_common_google_signin_btn_tint.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-xhdpi-v4_notification_bg_normal.9.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-si_values-si.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-ne_values-ne.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-xxhdpi-v4_common_google_signin_btn_text_dark_normal_background.9.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-xhdpi-v4_common_google_signin_btn_icon_light_normal_background.9.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-it_values-it.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-de_values-de.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/layout-v21_notification_action.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-ta_values-ta.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-es_values-es.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/layout_notification_template_big_media_custom.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable_notification_bg.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-mdpi-v4_googleg_disabled_color_18.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-el_values-el.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable_common_google_signin_btn_text_dark_focused.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-hdpi-v4_notification_bg_low_pressed.9.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-xhdpi-v4_common_google_signin_btn_icon_dark_normal_background.9.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-mn_values-mn.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/layout-v16_notification_template_custom_big.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-eu_values-eu.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable_notification_tile_bg.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-cs_values-cs.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-nb_values-nb.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-mdpi-v4_notification_bg_low_normal.9.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-az_values-az.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-v16_values-v16.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable_common_google_signin_btn_text_light_normal.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-be_values-be.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable_common_google_signin_btn_text_dark.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-mdpi-v4_common_google_signin_btn_text_dark_normal_background.9.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-sw_values-sw.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable_common_google_signin_btn_icon_light_focused.xml.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-mdpi-v4_notification_bg_normal.9.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-xhdpi-v4_common_google_signin_btn_text_light_normal_background.9.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-tr_values-tr.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-mdpi-v4_notification_bg_low_pressed.9.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-or_values-or.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-sq_values-sq.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-kn_values-kn.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-lt_values-lt.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-hdpi-v4_notification_bg_low_normal.9.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/values-ja_values-ja.arsc.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable-xhdpi-v4_notification_bg_normal_pressed.9.png.flat\
          -R\
          /Users/vagrant/git/build/sms_autofill/intermediates/res/compiled/release/drawable_common_google_signin_btn_icon_light.xml.flat\
          --auto-add-overlay\
          --non-final-ids\
          -0\
          apk\
          --no-version-vectors
  Daemon:  AAPT2 aapt2-3.2.1-4818971-osx Daemon #0
* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
* Get more help at https://help.gradle.org
BUILD FAILED in 2m 7s
Running Gradle task 'assembleRelease'...                          129.3s (!)
*******************************************************************************************
The Gradle failure may have been because of AndroidX incompatibilities in this Flutter app.
See https://goo.gl/CP92wY for more information on the problem and how to fix it.
*******************************************************************************************
Gradle task assembleRelease failed with exit code 1
Failed to build Android platform, error: exit status 1
|                                                                              |
+---+---------------------------------------------------------------+----------+
| x | flutter-build@0.9.2 (exit code: 1)                            | 363 sec  |
+---+---------------------------------------------------------------+----------+
| Issue tracker: ...thub.com/bitrise-steplib/bitrise-step-flutter-build/issues |
| Source: https://github.com/bitrise-steplib/bitrise-step-flutter-build        |
+---+---------------------------------------------------------------+----------+
```

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -8,7 +8,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.2.1'
+        classpath 'com.android.tools.build:gradle:3.3.0'
     }
 }
 
@@ -22,7 +22,7 @@ rootProject.allprojects {
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 27
+    compileSdkVersion 28
 
     defaultConfig {
         minSdkVersion 16

